### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by DcRonan**
### Interaction

1. Created Event Handlers
2. Implemented working functionality to display and submit numbers and operations

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi @DcRonan :wave: 

### Your project is coming along just fine, but there is a bug :bug: 

 After performing an operation, when I use the **A/C** button to clear the screen, other operations don't seem to work properly except add operation.

Please do look into it; use this gif below for reference.



illustration

#### That's it make this update then submit another form for a review.

### Have a great day :slightly_smiling_face: 

### ``` Uchechukwu Henry Inyama ```

**Comment by DcRonan** [comment]
Hi guys, run 'npm start' and you will see that it works.

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    case '%':
+      if (next) {
+        total = operate(total, next, operation);
+        total = operate(total, '100', '÷');
+        next = null;
+        operation = null;
+      } else {
+        total = operate(total, '100', '÷');
+      }
+      break;
```
Comment (christianotieno) [inline src/logic/calculate.js:20]: ## Fix `%` operator functionality.
- [ ] During a new operation, when I click on a number and then the `%` operator, `React` throws the following error. 👇 

calculator-demo

- However, on the positive though, after an operation, the operation works just fine. Kindly attend to this before we proceed.

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @DcRonan 👋 
Thank you for working on the suggested changes. 👍 

Kindly take note of the following suggestion based on a bug that might have been introduced during your working on the requested changes above.
### Happy coding 💻 
#### Christian Otieno (CR)

**Review by christianotieno — APPROVED** [review]
### Milestone Approved :champagne:
Hello @DcRonan 👋  Congratulations. You got it right this time.
 ##
This milestone has met all the requirements needed for approval. 👏 
Congratulations on the hard work done on this milestone. Good job 👍 
I will go ahead and mark this as complete. Good luck with the next tasks 🥇 
##
### Happy coding 💻 
#### Christian Otieno (CR)

**Pull request merged**